src/khmer/_cpy_graphlabels.cc

```diff
@@ -391,6 +397,30 @@ labelhash_get_tag_labels(khmer_KGraphLabels_Object * me, PyObject * args)
 
 
 PyObject *
+labelhash_link_tag_and_label(khmer_KGraphLabels_Object * me, PyObject * args)
+{
+    LabelHash * labelhash = me->labelhash;
+
+    PyObject * tag_o;
+    HashIntoType tag;
+    Label label;
+
+    if (!PyArg_ParseTuple(args, "OK", &tag_o, &label)) {
+        return NULL;
+    }
+    if (!ht_convert_PyObject_to_HashIntoType(tag_o, tag,
+            labelhash->graph)) {
+        return NULL;
+    }
+
+    labelhash->link_tag_and_label(tag, label);
+
+    Py_INCREF(Py_None);
+    return Py_None;
```

Comment:
What is the purpose of these two lines? Reading between the lines, it looks like its simply making the default behavior explicit. @ctb